devbox.d/php/php.ini

```diff
@@ -0,0 +1,10 @@
+# Lagoon has less memory, but we need more for development environments because
+# Drupal installation takes lots of RAM.
+memory_limit = 256M
+
+# Xdebug.
+# Uncomment the first line to enable xdebug. And do NOT commit the change.
+;xdebug.mode=debug
```

Comment:
FYI: The default xdebug mode has been changed in https://github.com/AmazeeLabs/silverback-template/commit/9093d75bcf808219e7d07e216642d2e631e3ae0e